src/app/clusters/group-key-mgmt-server/group-key-mgmt-cluster.cpp

```diff
@@ -102,211 +96,213 @@ struct GroupTableCodec
     }
 };
 
-class GroupKeyManagementAttributeAccess : public AttributeAccessInterface
+struct KeySetReadAllIndicesResponse
```

Comment:
please add a doccomment explaining what this struct is intended to do and why it exists.